src/hexo/themes/sonarwhal/layout/partials/scan-result.hbs

```diff
@@ -108,7 +108,14 @@
                         <div class="module module--primary results">
                             {{#each categories as |category|}}
                             <section class="rule-result" id="{{name}}">
-                                <h3>{{name}}</h3>
+                                <div class="rule-result--category">
+                                    <h3>{{name}}</h3>
+                                    {{#if (isFinish ../status)}}
+                                        {{#unless (noIssue category)}}
+                                        <button title="expand" class="button-expand-all closed">+ Expand All</button>
```

Comment:
In order to be consistent with what is used in JavaScript): `Expand` => `expand`